### ISSUE TITLE
NFR: Change Truck Configurator iFrame Integration to Enhanced Embed Solution

### ISSUE THREAD

**Issue by ZoeBeykirch** [aemsites/volvotrucks-na#53]
### Context/Description

As a Volvo Trucks website owner, I want to replace the current iFrame integration for the truck configurator with the improved embed solution, so that we can enhance loading performance and have more analytics capabilities, replicating the successful embed implementation used for the Mack Trucks launch.

### Acceptance Criteria

**User AC:**
- [x] Ensure that all configurator functionalities and user settings are preserved and fully operational.

**Non-functional AC:**
- [x] Replace the existing iFrame integration with the new embed solution for USA and Canadian sites:
   -  https://www.volvotrucks.us/truck-builder
   - https://www.volvotrucks.ca/en-ca/truck-builder
   - https://www.volvotrucks.ca/fr-ca/truck-builder

### Out of scope

These will be handled separately: 
- Performance loading times benchmarks should meet or exceed those achieved during the Mack Trucks launch.
- GA4 analytics tracking should work with the new embed solution to monitor user interactions.

### Testing Criteria 

- [x] Check that the configurator still works for VT 

### Note

The embed solution should incorporate the same improvements and features successfully implemented during the Mack Trucks launch:
https://github.com/hlxsites/vg-macktrucks-com/issues/579
https://github.com/hlxsites/vg-macktrucks-com/issues/591

**Comment by amglebihan** [comment]
The code changes were completed in a previous sprint. What remained in this sprint, and unblocked the ticket, was Kalle's work in resolving some issues, Girish's testing, and I updated how the configurator is authored across the three markets.
